apps/main-app/src/containers/allelePage/AlleleToDiseaseTable.js

```diff
@@ -4,21 +4,28 @@ import {
   EvidenceCodesCellCuration,
   ReferencesCellCuration,
 } from '../../components/dataTable';
-import AlleleSource from '../../components/dataTable/AlleleSource';
```

Comment:
was there not a AlleleSourceCuration file that does this?

Reply:
AlleleSource only formats the link, does not consolidate the data coming form the endpoing